File: DSA/Java/Q4.java
public class Q4 {
    public static int compress(char[] chars) {
        //write your code here
    }

    public static void main(String[] args) {
        char[] chars = {'a', 'a', 'b', 'b', 'c', 'c', 'c'};
        int newSize = compress(chars);
        System.out.println("New length: " + newSize);
        for (int i = 0; i < newSize; i++) {
            System.out.print(chars[i] + " ");
        }
        System.out.println();
    }
}

File: DSA/Python/Q4.py
def compress(chars):
   #write your code here

if __name__ == "__main__":
    chars = ['a', 'a', 'b', 'b', 'c', 'c', 'c']
    newSize = compress(chars)
    print("New length:", newSize)
    print(chars[:newSize])
